docs/source/register_file.rst

```diff
@@ -70,12 +70,12 @@ For more information regarding the clock gating cell, checkout :ref:`getting-sta
 FPU Register File
 -----------------
 
-In case the optional FPU is instantiated, the register file is extended
+If the optional FPU is instantiated, unless ZFINX is configured, the register file is extended
 with an additional register bank of 32 registers ``f0``-``f31``. These registers
 are stacked on top of the existing register file and can be accessed
 concurrently with the limitation that a maximum of three operands per
 cycle can be read. Each of the three operands addresses is extended with
-an fp_reg_sel signal which is generated in the instruction decoder
+an ``fp_reg_sel`` signal which is generated in the instruction decoder
```

Comment:
this signal does not exist, maybe we can say "extended with a register file select signal which ..."

Reply:
yes - it's not wise to refer to signal names in the text if it can be avoided